examples/mobile/ScheduleApp/.tproject

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Is this file needed?
I think it's no needed for examples.

Reply:
@singa2000 I requested this from Kornelia in previous PR. All exmples have sample .tproject file so we can test them on emulator.